src/coll_score/ucc_coll_score_map.c

```diff
@@ -193,10 +211,18 @@ void ucc_coll_score_map_print_info(const ucc_score_map_t *map)
                                        sizeof(range_str));
                 ucc_score_to_str(range->super.score, score_str,
                                  sizeof(score_str));
+#if ENABLE_DEBUG == 1
```

Comment:
maybe pass log level to this function and instead of adding conditional compilation print info when log level is debug?